src/serve/server.ts

```diff
@@ -105,8 +106,11 @@ let ownerSizeTotalWorker: WorkerType | undefined
 let creditsWorker: WorkerType | undefined
 
 try {
-  ownerSizeTotalWorker = createWorker(join(__dirname, 'ownerSizeTotalWorker.ts'))
-  creditsWorker = createWorker(join(__dirname, 'creditsWorker.ts'))
+  const workerExtension = existsSync(join(__dirname, 'serve', 'ownerSizeTotalWorker.js')) ? '.js' : '.ts'
+  const workerDir = workerExtension === '.js' ? join(__dirname, 'serve') : __dirname
+
+  ownerSizeTotalWorker = createWorker(join(workerDir, `ownerSizeTotalWorker${workerExtension}`))
+  creditsWorker = createWorker(join(workerDir, `creditsWorker${workerExtension}`))
```

Comment:
Thanks for fixing the bundling, but can you please try to do this more cleanly? You can, for example, use `define` in `build.ts` to know, for certain and without a runtime check, what the path should be.

For example,

```
const workerDir = IS_BUNDLE ? join(__dirname, 'serve') : __dirname

ownerSizeTotalWorker = createWorker(join(workerDir, `ownerSizeTotalWorker${workerExtension}`))
```

But actually, I suspect that the way things are now, the `.js` version of the file will always exist, which makes all of these checks redundant. So, you can just have:

```
  ownerSizeTotalWorker = createWorker(join(__dirname, 'serve', 'ownerSizeTotalWorker.js'))
  creditsWorker = createWorker(join(__dirname, 'serve', 'creditsWorker.js'))
```